test/integration/patients/patient/patient-flow.js

```diff
@@ -228,6 +241,33 @@ context('patient flow page', function() {
       .get('.sidebar')
       .find('[data-duration-region]')
       .should('contain', '20 mins');
+
+    cy.sendWs({
+      category: 'CommentAdded',
+      resource: {
+        type: 'patient-actions',
+        id: testFlowAction.id,
+      },
+      payload: {
+        comment: {
+          type: 'comments',
+          id: testSocketComment.id,
+        },
+        attributes: {
+          message: 'New websocket comment.',
```

Comment:
If we use this current implementation, this portion of the ws notification message isn't used.